com.cubrid.cubridmigration.core/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" 
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.cubrid.cubridmigration</groupId>
+    <artifactId>cubridmigration</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../</relativePath>
+  </parent>
+  <artifactId>com.cubrid.cubridmigration.core</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>eclipse-plugin</packaging>
+  
+  <build>
+    <sourceDirectory>src/</sourceDirectory>
+  </build>
+
+  <profiles>
+    <profile>
+        <id>desktop</id>
+    </profile>
+    
+    <profile>
+        <id>console</id>
+        <activation>
+            <property>
+                <name>consoleBuild</name>
+                <value>true</value>
+            </property>
+        </activation>
+
+        <build>
+            <resources>
+              <resource>
+                <directory>src/</directory>
+                <includes>
+                  <include>**/*.properties</include>
+                  <include>**/*.xml</include>
+                </includes>
+              </resource>
+            </resources>
+
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-jar-plugin</artifactId>
+                    <version>2.3.2</version>
+                    <configuration>
+                        <archive>
+                            <manifest>
+                                <addClasspath>true</addClasspath>
+                                <classpathPrefix>lib</classpathPrefix>
+                            </manifest>
+                        </archive>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </build>
+        
+        <dependencies>
+            <dependency>
+                <groupId>com.cubrid.cubridmigration</groupId>
+                <artifactId>com.cubrid.common.configuration</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+            </dependency>
+            
+            <dependency>
```

Comment:
In fact, the groupId is different and there are versions that the maven repository does not currently support.
The list is as in the attached file.
[dependency.patch](https://github.com/CUBRID/cubrid-migration/files/14307913/dependency.patch)

I think currently using the file in cubrid-migration/com.cubrid.cubridmigration.core/lib.
So first, only modify the warning message.
Afterwards I hope to modify it so that the dependency lib can be used in tycho in the future.

Reply:
Only the console version was currently using the Maven repository, but let's use the file located in cubrid-migration/com.cubrid.cubridmigration.core/lib.
We will make changes to both desktop and console when we make changes to make dependency lib available in tycho in the future.

Reply:
Yes, as you said, your plan and mine are the same.
Please use the dependency lib for tycho later.

But It now contains a WARNING message and is not used fineddlly so it needs to be modified.
For reference, the patch file can be applied using command 'git apply dependency.patch' from source root.

- WARNING LOG -
[WARNING] The POM for com.cubrid.cubridmigration:xmlbeans:jar:2.3.0 is missing, no dependency information available
[WARNING] The POM for com.cubrid.cubridmigration:jxl:jar:11.2 is missing, no dependency information available
[WARNING] The POM for com.cubrid.cubridmigration:poi:jar:3.8-20120326 is missing, no dependency information available
[WARNING] The POM for com.cubrid.cubridmigration:poi-ooxml:jar:3.8-20120326 is missing, no dependency information available
[WARNING] The POM for com.cubrid.cubridmigration:stax2-api:jar:3.1.1 is missing, no dependency information available
[WARNING] The POM for com.cubrid.cubridmigration:woodstox-core-lgpl:jar:4.1.2 is missing, no dependency information available
[WARNING] The POM for com.cubrid.cubridmigration:opencsv:jar:2.3 is missing, no dependency information available

Reply:
I've tried making changes to use the library under com.cubrid.cubridmigration.core/lib when building the console version, but I'm having trouble finding the library.
First, I'll fix warning and modify it to use the dependency library for tycho later.